Airports/EGQL/Positions.txt

```diff
@@ -1,4 +1,4 @@
-EGQL_APP:Leuchars Approach:126.500:QLR:A:EGQL:APP:-:-:0201:0247:N056.22.23.000:W002.52.02.000
+EGQL_APP:Leuchars Approach:126.505:QLR:A:EGQL:APP:-:-:0201:0247:N056.22.23.000:W002.52.02.000
```

Comment:
This is the approach frequency definition, is that supposed to be the same as Zone? They seem to have updated the zone, but not approach frequency in the AIP? @AliceFord 
![image](https://github.com/VATSIM-UK/UK-Sector-File/assets/61326713/ccfef2d7-f856-4739-a234-55fdafea64b1)
I think what you've done here @Lucak1011 is probably correct, just asking the question.

Reply:
it will be the same I suspect AIDU have just done their usual thing and messed it up.